kalamine/generators/klc.py

```diff
@@ -81,6 +81,8 @@ def klc_virtual_key(layout: "KeyboardLayout", symbols: list, scan_code: str) ->
         return "OEM_MINUS"
     elif base == " ":
         return "SPACE"
+    elif base == "-1" and shifted == "-1":
+        raise Exception("Empty key are not allowed, please complete the description")
```

Comment:
```suggestion
        raise Exception("Empty keys are not allowed, please complete the description")
```

Reply:
This section confusing. Where do these "-1" come from? This doesn’t seem sane to me. :-)

The exception message could be more self-explanatory. What should be completed?
And while we’re at it, the "too many OEM keys" message could be a bit better as well.

Reply:
I don’t know where the `-1` comes from, isn’t it filled by the `toml` parser?

"Too many OEM" is not very clear indeed, do you have a better idea? I don’t have any ^^'